src/views/document/utils/boxes/ElevationProfile.vue

```diff
@@ -24,10 +24,17 @@
     </form>
 
     <div ref="graph" class="elevation-profile-chart" />
+
+    <div class="elevation-profile-fullscreen-toggle" @click="toggleFullScreenProfile">
+      <fa-icon :icon="toggleIcon" />
+    </div>
```

Comment:
Add `title` attribute to help users understanding the goal of the button?

Reply:
see https://github.com/c2corg/c2c_ui/pull/2232/commits/e873c860dc7a2905e3044f9892ad8d816b566d18